src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -76,6 +77,8 @@ internal ReturnTypeAndDiagnostics(TypeSymbol returnType, ImmutableArray<Diagnost
                 DeclarationModifiers.Static |
                 syntax.Modifiers.ToDeclarationModifiers();
 
+            this.IsUnsafe = inUnsafeRegion || _declarationModifiers.HasFlag(DeclarationModifiers.Unsafe);
```

Comment:
>_declarationModifiers.HasFlag(DeclarationModifiers.Unsafe) [](start = 46, length = 58)

Can local functions be marked unsafe? It doesn't look like there are any tests for that scenario.